src/components/ProductCard/ProductCard.tsx

```diff
@@ -0,0 +1,31 @@
+import type { FC } from 'react';
+
+import IconCart from '@icons/cart.svg?react';
+import type { Product } from '@interfaces/Product';
+
+import styles from './ProductCard.module.css';
+
+interface ProductCardProps {
+    product: Product;
+}
+
+const ProductCard: FC<ProductCardProps> = ({ product }) => (
+    <article className={styles.card}>
+        <div className={styles.cardImageWrapper}>
+            <img className={styles.cardImage} src={product.images[0]} alt={product.title} />
+        </div>
+        <div className={styles.cardBody}>
+            <h2 className={styles.cardTitle}>{product.title}</h2>
+            <div className={styles.cardFooter}>
+                <span className={styles.cardPrice}>
+                    <span className={styles.cardPriceValue}>{product.price}</span>₴
+                </span>
+                <button className={styles.cardCartBtn}>
```

Comment:
Do you plan to implement this:
> When the user clicks on Cart icon inside a ProductCard, we need to make the icon active.
Also, we need to show counter of added items in header cart icon.

It was not described in AC, so you can skip it for now😏 But, I think, it could be a nice practice for you.